socialhome/streams/app/tests/components/StreamElement.tests.js

```diff
@@ -87,14 +89,14 @@ describe("StreamElement", () => {
                 // Actual thing we are testing - the share
                 target.instance().share()
 
-                moxios.wait(function() {
+                moxios.wait(() => {
```

Comment:
Sorry I changed two functions to arrow functions but the wrong ones I realized after merging my PR - thanks for doing these :D